css/DesktopVersion.css

```diff
@@ -0,0 +1,417 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;1,100;1,200;1,300;1,400&display=swap");
+
+* {
+  font-family: "Poppins", sans-serif;
+}
+
+@media only screen and (min-width: 768px),
+  (min-width: 1024),
+  (min-width: 1440px) {
+  body {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    background-color: #dfe1e6;
+  }
```

Comment:
- [ ] Great work implementing your media query 💯. However, please kindly ensure that your mobile version remains stable below 768px. Currently, when I go below 768px a lot of styles are not being applied 🤓.

Here you can see a video showing what happens when I go below 768px 👇🏽


https://user-images.githubusercontent.com/116701485/236355479-acca8bda-9065-46b0-a728-e943fabaa719.mp4